src/panels/lovelace/common/format-lovelace-config.ts

```diff
@@ -0,0 +1,17 @@
+import { LovelaceConfig } from "../../../data/lovelace";
+
+export const formatLovelaceConfig = (
```

Comment:
We might not need this if we can just pass viewIndex and cardIndex to the edit button for cards.

The edit card dialog then gets params `(config: LovelaceConfig, [viewId, cardId])`